Makefile

```diff
@@ -301,6 +301,10 @@ $(1)build/Release/.buildstamp: $(1)build/Makefile $(2) $(ADDON_PREREQS)
 	$(NODE_GYP) --directory=$(1) build
 	@touch $$@
 $(1)build/Release/binding.node: $(1)build/Release/.buildstamp
+$(1)build/Debug/.buildstamp: $(1)build/Makefile $(2) $(ADDON_PREREQS)
+	$(NODE_GYP) --directory=$(1) --debug build
+	@touch $$@
+$(1)build/Debug/binding.node: $(1)build/Debug/.buildstamp
```

Comment:
Could we reduce duplication by using `$(BUILDTYPE)`?

Reply:
Wouldn't make much difference because you still need the rule that passes `--debug` to node-gyp.

You could unify them if the call to `node-gyp build` was replaced with `make -C $(1)/build` but that's somewhat unofficial.

Reply:
`$(NODE_GYP) --directory=$(1) --$(BUILDTYPE) build`?

Reply:
Doesn't work; the argument is case-sensitive.  I suppose you could use `$(subst $(subst ...), ...)` but that's arguably harder to read than ^.